extras/Build/CMake/JUCEUtils.cmake

```diff
@@ -1767,6 +1814,13 @@ function(_juce_set_fallback_properties target)
         _juce_set_property_if_not_set(${target} VST3_CATEGORIES Fx)
     endif()
 
+    # LV2_CATEGORIES
+    if(is_synth)
+        _juce_set_property_if_not_set(${target} LV2_CATEGORIES InstrumentPlugin)
+    else()
+        _juce_set_property_if_not_set(${target} LV2_CATEGORIES Plugin)
+    endif()
+
```

Comment:
I'm not sure we need these. Admittedly, I'm not super well versed in the JUCE LV2 wrapper, but I don't think there's any code that refers to `LV2_CATEGORIES` like VST3 needs. It gets its categories from other properties, like `IS_SYNTH`.